src/Form/form-autosuggest.mdx

```diff
@@ -69,6 +80,9 @@ Form auto-suggest enables users to manually select or type to find matching opti
 
 ```jsx live
 () => {
+    const [userProvidedText, setUserProvidedText] = useState('');
```

Comment:
It seems to me that this `mdx` file should have 2 indent spaces (currently 4). This can be seen by looking at other `mdx` files of the [Form component](https://github.dev/openedx/paragon/blob/716a6ddd932d61a5b5ab594e8e525c3dfffe20ab/src/Form/form-autosuggest.mdx).

Reply:
I fixed the indentation (by copying the jsx from the mdx file into a temp.jsx file and running lint fix)